release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1858,6 +1881,11 @@ The Ingress Operator was changed to consider unspecified values and default valu
 
 * Previously, there was no validation to prevent the Local Storage Operator from creating one device to multiple PVs. This release adds validation for this scenario so that trying to create a PV on a block device where one is already provisioned by the Local Storage Operator will fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1744385[*BZ#1744385*])
 
+*Insights Operator*
+
+* Previously, the Insights Operator could gather data for an unlimited number of certificate signing requests (CSRs) in a single report. This resulted in excessive data collection for clusters with many CSRs. Insights Operator now gathers data for a maximum of 5000 CSRs in a single report.
```

Comment:
s/CSRs. Insights Operator/CSRs. The Insights Operator

Reply:
Resolved.